### ISSUE TITLE
Error installing v1.22.1

### ISSUE THREAD

**Issue by faycito** [pedroslopez/whatsapp-web.js#2407]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

```
(node:21692) UnhandledPromiseRejectionWarning: Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'CallCollection')
    at __puppeteer_evaluation_script__:9:66
    at ExecutionContext._evaluateInternal (C:\Users\ismae\Documents\chat-pe-front\release\app\node_modules\puppeteer\src\common\ExecutionContext.ts:273:13)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ExecutionContext.evaluate (C:\Users\ismae\Documents\chat-pe-front\release\app\node_modules\puppeteer\src\common\ExecutionContext.ts:140:12)
    at async Client.initialize (C:\Users\ismae\Documents\chat-pe-front\release\app\node_modules\whatsapp-web.js\src\Client.js:278:9)
(node:21692) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag --unhandled-rejections=strict (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
```

Version Installed:
`    "whatsapp-web.js": "^1.22.1"`

Error just happening on Windows, tested in Mac and it works

### Expected behavior

just running :( 

### Steps to Reproduce the Bug or Issue

1.- Request the QR
2.- Link the device

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

OS: Windows
Phone: All devices
wweb-js version: 1.22.1

**Comment by BasLandman** [comment]
I have the same issue with initializing sessions. I can't use the code anymore

**Comment by EdsonIV** [comment]
> ### Is there an existing issue for this?
> 
>     * [x]  I have searched the existing issues
> 
> 
> ### Describe the bug
> 
> ```
> (node:21692) UnhandledPromiseRejectionWarning: Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'CallCollection')
>     at __puppeteer_evaluation_script__:9:66
>     at ExecutionContext._evaluateInternal (C:\Users\ismae\Documents\chat-pe-front\release\app\node_modules\puppeteer\src\common\ExecutionContext.ts:273:13)
>     at processTicksAndRejections (node:internal/process/task_queues:95:5)
>     at async ExecutionContext.evaluate (C:\Users\ismae\Documents\chat-pe-front\release\app\node_modules\puppeteer\src\common\ExecutionContext.ts:140:12)
>     at async Client.initialize (C:\Users\ismae\Documents\chat-pe-front\release\app\node_modules\whatsapp-web.js\src\Client.js:278:9)
> (node:21692) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag --unhandled-rejections=strict (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
> ```
> 
> Version Installed: ` "whatsapp-web.js": "^1.22.1"`
> 
> Error just happening on Windows, tested in Mac and it works
> ### Expected behavior
> 
> just running :(
> ### Steps to Reproduce the Bug or Issue
> 
> 1.- Request the QR 2.- Link the device
> ### Relevant Code
> 
> _No response_
> ### Browser Type
> 
> Chromium
> ### WhatsApp Account Type
> 
> Standard
> ### Does your WhatsApp account have multidevice enabled?
> 
> Yes, I am using Multi Device
> ### Environment
> 
> OS: Windows Phone: All devices wweb-js version: 1.22.1
> ### Additional context
> 
> _No response_

same problem

**Comment by EdsonIV** [comment]
same problem.

**Comment by qiraxyz** [comment]
are you sure the npm installation is finished?. or maybe u can try to delete package-lock.json and node modules and reinstall the package again. u can try reinstall again using stable npm v8 or v7.

**Comment by alechkos** [comment]
Fixed in #2385
